library/source/wrapper/DdUrlTransfer.bs

```diff
@@ -19,17 +19,27 @@ class DdUrlTransfer
     ' requests to
     ' @param tracingSamplingRate (double) The sampling rate to add
     ' trace headers to the request, between 0 and 100
+    ' @param tracedHosts (array) the hosts for which requests will have a trace generated
```

Comment:
minor - in other sdks we ended up having dictionary of hosts and supported tracing headers when introducing b3 and w3c.

It's probably out of scope of this PR, but might be a good idea to use dictionary already (and just support one type of tracing headers).

Reply:
that's a good point but indeed out of scope right now